app/features/build/views/build.erb

```diff
@@ -25,7 +25,11 @@
       <td>Job Trigger</td><td><%= build.trigger %></td>
     </tr>
     <tr>
-      <td>Sha</td><td><%= build.sha %></td>
+      <td>Sha</td><td>
+      <a href="<%= build.link_to_remote_commit %>" target="_blank" class="commit-sha">
+        <%= build.sha %>
```

Comment:
I'm basing in my experience with other CIs that, even in the build details, the whole sha is meaningful, so i think we should use the short form whenever is possible (even in GitHub is hard to see the full commit sha 😅 )

Reply:
Jenkins shows the full hash in the build details page, I would like to hear more opinions on this (or anyway it could be done in a separately PR), personally I don't have a strong opinion on what to show.